adage/analyze/tests.py

```diff
@@ -1025,3 +1034,107 @@ def test_expressionvalue_big_post(self):
             self.deserialize(resp)['meta']['total_count'],
             self.expressionvalue_count
         )
+
+
+@override_settings(HAYSTACK_CONNECTIONS=TEST_INDEX)
+class SearchIndexTestCase(ResourceTestCaseMixin, TestCase):
+    searchURI = '/api/v0/search/'
+    experiments = [
+        {
+            'accession':    'E-GEOD-31227',
+            'name':         'Expression data of Pseudomonas aeruginosa isolates'
+                            ' from Cystic Fibrosis patients in Denmark',
+            'description':  'CF patients suffer from chronic and recurrent '
```

Comment:
While this style was not strictly against PEP8 rules, it does fall squarely into the "pet peeve" category here (see last example): https://www.python.org/dev/peps/pep-0008/#pet-peeves .

Fixed.